lib/internal/repl.js

```diff
@@ -20,6 +20,8 @@ function _writeToOutput(repl, message) {
 }
 
 function createRepl(env, opts, cb) {
+  console.log(`Welcome to Node.js ${process.version}.\n` +
```

Comment:
I would put this in `lib/internal/main/repl.js`, prior to the creation of the repl, as this only affects the repl started from the command line.

https://github.com/nodejs/node/blob/c2d374fcccf04b6bf62acabe09647fb4fab2dab8/lib/internal/main/repl.js#L22-L25

Reply:
Move to `lib/internal/main/repl.js` seems like won't work with console.log https://travis-ci.org/gengjiawen/node/jobs/490981933.

Reply:
That seems strange, `createRepl` only gets called in `lib/internal/main/repl.js` and putting the console call before the function call or inside the function should not make a difference. Where did you put the log statement?

Reply:
https://github.com/nodejs/node/commit/8188e3544b623bb4f8cddff271b4c597713138b7#diff-e231c3a0a2b30483659fe7c9737c38d8R29

Reply:
That is too late - the repl has already been setup in the callback. This should be placed before the `createInternalRepl` call, in the outer scope.